Models/ApiCheck.cs

```diff
@@ -1,10 +1,14 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using System.Net;
 
 namespace Models; 
 
 public class ApiCheck {
+    public const HttpStatusCode ClientSideFailure = (HttpStatusCode) 1023;
+    public const HttpStatusCode InvalidWseConfiguration = (HttpStatusCode) 1024;
```

Comment:
This is kinda hacky, preferable would be a discriminated union, but that's not a thing in C#!